//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// abc_O_P.h
//
// Code generation for function 'abc_O_P'
//

#ifndef ABC_O_P_H
#define ABC_O_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void abc_O_P(double P_init, double V_init, double A_init, double P_wayp,
             double V_max, double A_max, double J_max, double J_min,
             creal_T t[21]);

#endif
// End of code generation (abc_O_P.h)
